doc/classes/ScrollBar.xml

```diff
@@ -23,6 +23,22 @@
 		</signal>
 	</signals>
 	<theme_items>
+		<theme_item name="padding_bottom" data_type="constant" type="int" default="0">
+			Padding between the bottom of the [theme_item scroll] element and the [theme_item grabber]. Only used in [HScrollBar].
```

Comment:
Why are the items bound to ScrollBar if they are only relevant for one direction?